### DISCUSSION TITLE
Module not found: Can't resolve '@keystone-6/core/context'

### DISCUSSION THREAD

**Discussion by salvatorericcardi** [keystonejs/keystone#8011] [Questions] [answered]
### **How to reproduce the problem**
Simply import @keystone-6/core/context module and you will have a module not found error

### **What I'm expecting**
I need to use getContext to bring content on frontend

### **The error**
![error](https://user-images.githubusercontent.com/29570607/196109219-52b275c5-6ccc-49a5-b0c5-a43a96422f8b.png)

### **Contextual informations**
Node version 16.15.0
Browser Google Chrome

**Comment by gautamsi** [comment, marked as answer]
next major update is not released which contains `getContext` code. wait for next major release

**Reply by dcousens** [reply]
Hi @riccardi1997,
As mentioned by @gautamsi, this is part of our next major release; that should be published this week.
I'm wondering where you learnt about the `@keystone-6/core/context` import though, as our documentation shouldn't update until the release is published.

Maybe you were browsing our examples on the `main` branch?

**Reply by salvatorericcardi** [reply]
Yes. In the while, is there an alternative to `getContext`? So I can continue coding until next major release.

**Reply by dcousens** [reply]
@riccardi1997 no alternatives that I can recommend, at least nothing other than the [latest major release](https://github.com/keystonejs/keystone/releases/tag/2022-10-19) now published :ship:

**Reply by jj-matos** [reply]
This was not solved with that major release. The issue persists.

**Reply by dcousens** [reply]
@jj-matos please see https://github.com/keystonejs/keystone/blob/main/examples/script/script.ts#L1 for how you might use `import { getContext } from '@keystone-6/core/context'` in your code
